package.json

```diff
@@ -31,6 +31,6 @@
     "uglify-js": "~2.4.15",
     "browserify": "5.11.1",
     "pbind": "~0.0.1",
-    "testem": "0.6.18"
+    "testem": "1.7.1"
```

Comment:
Bumped the `testem` version because `0.6.18` would hang when running `testem ci`. Since the current instructions are just to `npm install -g testem`, I updated to the latest (since that's what `<-` that would do)